workers/event_catcher/event_catcher.rb

```diff
@@ -18,17 +18,18 @@ def run!
 
     notify_started
 
-    logger.info("Collecting events...")
+    log_prefix = "MIQ(Vmware::InfraManager::EventCatcher)".freeze
+    logger.info("#{log_prefix} Collecting events...")
```

Comment:
NOTE if this worker gets more complicated we should build a Logger class that handles the prefix "automatically" but for now we only log in a single method so this seemed sufficient for now

Reply:
Nope this is only done here https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/logging.rb#L14-L35 as far as I know.

Reply:
The goal was to move to log_decorator, but that work never completed.